spec/db.spec.ts

```diff
@@ -12,6 +14,11 @@ import type { Server, ServerConfig } from '../src/server';
 
 chai.use(chaiAsPromised);
 
+const isLowerThanNode12 =
+  process.version.startsWith('v10') ||
+  process.version.startsWith('v9') ||
+  process.version.startsWith('v8');
```

Comment:
Same comment as the other PR, the lowest allowed version for this package is 10.13, so do not need to worry about `v9` or `v8`.

Reply:
I have fixed it in the last PR